### ISSUE TITLE
Nautilus extension crashes with unknown emblem

### ISSUE THREAD

**Issue by ckamm** [owncloud/client#4665]
I had nautilus running while adding a new sync folder and then saw this message in the nautilus command line output:
```
Share file /home/kamm/ownCloud/test2
Server response: SHARE:OK:/home/kamm/ownCloud/test2
Server response: STATUS:IGNORE:/home/kamm/ownCloud/.csync_journal.db
Server response: STATUS:IGNORE:/home/kamm/ownCloud/
Server response: REGISTER_PATH:/home/kamm/ownCloud2/
Server response: STATUS:NONE:/home/kamm/ownCloud2/
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gi/overrides/GLib.py", line 682, in <lambda>
    func_fdtransform = lambda _, cond, *data: callback(channel, cond, *data)
  File "/home/kamm/.local/share/nautilus-python/extensions/syncstate.py", line 122, in _handle_notify
    self._handle_server_response(l)
  File "/home/kamm/.local/share/nautilus-python/extensions/syncstate.py", line 135, in _handle_server_response
    listener(action, args)
  File "/home/kamm/.local/share/nautilus-python/extensions/syncstate.py", line 252, in handle_commands
    emblem = Emblems[newState]
KeyError: 'NONE'
```

It looks like we sometimes send a `NONE` state to the shell extension, but that state has no defined icon.

**Comment by dragotin** [comment]
I know what that is. I will fix that.

**Comment by ckamm** [comment]
Addressed in 15988c6fda5498cb1b1871e802626a55fecde9a2

**Comment by mcastroSG** [comment]
Hi hi:

I´m not able to reproduce it at current desktop client version, still some test has been test to check everything works fine.

### Test 1

**Steps Executed**

1.- Add new Sync Shared folder

**Results**

Everything seems to work fine, new folder is added and there is no error trace

### Test 2

**Steps Executed**

1.- Remove Sync Shared folder

**Results**

Everything seems to work fine, new folder is removed  and there is no error trace


Server OS: Ubuntu
Server Version: 9.1.0 Alpha

Client OS: Ubuntu 14.04 LTE
Client Version: Version 2.2.0nightly2016511 (build 1602)
